vectorsearch/indices/faiss-index.json

```diff
@@ -16,6 +16,11 @@
     "mappings": {
       "dynamic": "strict",
       "properties": {
+        {%- if id_field_name is defined and id_field_name != "_id" %}
```

Comment:
The `{%-` syntax can probably be replaced by `{%` in this context.
Please add this workload parameter to the README for completeness.
Likewise, please add information on the new workload parameter files there as well.

Reply:
Thanks for review @gkamat . I update PR. Can you please take a look at updated PR? Thanks.